### ISSUE TITLE
 Plugin panics when trying to fetch emojis

### ISSUE THREAD

**Issue by kostya9** [harrisoncramer/gitlab.nvim#389]
## Prerequsities

- [x] I'm on the latest version of the plugin
- [x] I've installed the required dependencies
- [x] I've run `:h gitlab.nvim.troubleshooting` and followed the steps there

## Setup Configuration and Environment

```
return {
	"harrisoncramer/gitlab.nvim",
	dependencies = {
		"MunifTanjim/nui.nvim",
		"nvim-lua/plenary.nvim",
		"sindrets/diffview.nvim",
		"stevearc/dressing.nvim", -- Recommended but not required. Better UI for pickers.
		"nvim-tree/nvim-web-devicons" -- Recommended but not required. Icons in discussion tree.
	},
	enabled = true,
	build = function() require("gitlab.server").build(true) end, -- Builds the Go binary
	config = function()
		require("gitlab").setup({
			debug = {
				request=true,
				response=true,
				gitlab_request=true,
				gitlab_response=true,
			},
			log_path = "D:/gitlab.nvim.log"

		})
	end,
}
```

OS: Windows

## Bug Description

Plugin panics when trying to fetch emojis. 

My hypothesis: it uses the wrong working directory.

## Reproduction Steps

1. Check out the feature branch
2. Open Neovim
3. Use <leader>gls
4. Observe error
```
gitlab.nvim: panic: could not find emojis at ./config/emojis.json                                                                        goroutine 1 [running]:                                                                                                                   github.com/harrisoncramer/gitlab.nvim/cmd/app.CreateRouter(0xc000108060, 0x1?, 0xc000193d88, {0xc000193df0, 0x3, 0x6734a7?})             ^IC:/Users/KostiantynSharovarsk/AppData/Local/nvim-data/lazy/gitlab.nvim/cmd/app/server.go:94 +0x3b4b                                    github.com/harrisoncramer/gitlab.nvim/cmd/app.StartServer(0xc000108060, 0xc000236588, {{0xc000124000, 0x3c}, {                           gitlab.nvim: 0xc00012401a, 0xe}, {0xc000124029, 0x13}, {0xc00014c558, 0x17}})                                                            ^IC:/Users/KostiantynSharovarsk/AppData/Local/nvim-data/lazy/gitlab.nvim/cmd/app/server.go:26 +0x156                                     main.main()                                                                                                                              ^IC:/Users/KostiantynSharovarsk/AppData/Local/nvim-data/lazy/gitlab.nvim/cmd/main.go:45 +0x38d                                           gitlab.nvim: Golang gitlab server exited: job_id: 3, exit_code: 1
```

## Screenshots

![image](https://github.com/user-attachments/assets/7d3ef6d2-decf-4bec-8291-40cf4fc534b7)

---

Note: if I add the config folder with emojis to my current working directory (NOT the directory of the plugin, but the repo I'm currently working in) -> it starts working.

**Comment by harrisoncramer** [comment]
I'm guessing this is related to the issue where the plugin is not building properly in your other issue, let's debug that one first and come back to this. Thanks for your patience!

**Comment by harrisoncramer** [comment]
Fixed by #419
